src/routes/index.js

```diff
@@ -1,15 +1,15 @@
 const express = require("express");
 
 const { authRouter } = require("./auth.router");
-const {productRouter} = require("./product.router");
-const {orderRouter} = require('./order.router');
+const { cartRouter } = require("./cart.router");
+const { productRouter } = require("./product.router");
+const { orderRouter } = require("./order.router");
 
 const routes = express.Router();
 
 routes.use("/auth", authRouter);
-routes.use('/products', productRouter);
-routes.use('/orders' , orderRouter);
+routes.use("/products", productRouter);
+routes.use("/cart", cartRouter);
+routes.use("/orders", orderRouter);
```

Comment:
```suggestion
routes.use("/products", productRouter);
routes.use("/carts", cartRouter);
routes.use("/orders", orderRouter);
```

다른 엔드포인트들과 컨벤션을 맞춰주세요